test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
@zer0 I think here or elsewhere we should test what happens with the cuddlefish loader when the widget module is required and make sure that what we want to see happen actually happens.  Since we know there will be plenty of people that hit this use case.